packages/manager/modules/pci/src/projects/new/payment/components/default/index.html

```diff
@@ -4,27 +4,55 @@
         data-translate="pci_project_new_payment_default_title"
     ></h2>
 
-    <p
-        data-translate="pci_project_new_payment_default_explain"
-        data-translate-values="{ paymentMehodSection: $ctrl.sectionUrls.payment, myAccountSection: $ctrl.sectionUrls.myAccount }"
+    <p class="text-justify"
+       data-translate="pci_project_new_payment_default_explain_info"
     ></p>
 
+
+    <!--Default credit card -->
     <div class="default-method">
-        <img
-            class="d-inline-block vertical-middle"
-            data-ng-if="$ctrl.defaultPaymentMethod.icon.data"
-            data-ng-src="{{ $ctrl.defaultPaymentMethod.icon.data }}"
-            alt=""
-        />
-        <span
-            class="d-inline-block vertical-middle"
-            data-ng-bind="$ctrl.defaultPaymentMethod.label"
-        >
-        </span>
-        <strong
-            class="text-success d-inline-block vertical-middle"
-            data-translate="pci_project_new_payment_default_method"
-        >
-        </strong>
+
+        <!--Go to payment section-->
+        <p class="text-right">
+            <a data-ng-href="{{::$ctrl.sectionUrls.payment}}"
+               target="_blank"
+            >
+                <span data-translate="pci_project_new_payment_default_method_use_others_payment_methods"
+                >
+                </span>
+                <span class="oui-icon oui-icon-external-link"
+                      aria-hidden="true"
+                ></span>
+            </a>
+        </p>
+
+        <!--Default payment method-->
+        <div class="p-3 border border-info rounded d-flex align-items-center justify-content-between">
+
+            <!--Logo-->
+            <img data-ng-if="$ctrl.defaultPaymentMethod.icon.data"
+                 data-ng-src="{{ $ctrl.defaultPaymentMethod.icon.data }}"
+                 alt=""
```

Comment:
**question:** Can we add some text to `alt`?

Reply:
The `img` element was already here.
I juste retake it.

I will ask to know what we can add.

Reply:
Done